src/components/transactions/TxDetails/index.tsx

```diff
@@ -76,7 +76,7 @@ const TxDetailsBlock = ({ txSummary, txDetails }: TxDetailsProps): ReactElement
       {/* /Details */}
       <div className={`${css.details} ${isUnsigned ? css.noSigners : ''}`}>
         {isOrderTxInfo(txDetails.txInfo) && (
-          <div>
+          <div className={css.swapOrder}>
```

Comment:
The class exists in the CSS, right? Because that PR also removed it from there.

Reply:
It only removed it for the border top property but the rest is still applied.